### PR TITLE
Updating .tt file due to renaming of LbfgsPoissonRegression.

### PR DESCRIPTION
**Pull request by zeahmed**
This PR fixes the issue related to renaming done in #3034. Currently, we are finding a way to forcing people to update the .tt file instead of .cs file generated from .tt.

### CODE REVIEW

**Comment by abgoswam** [comment]
>         ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PoissonRegressionWithOptions.cs)]

should we fix this as well ?

---
Refers to: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs:596 in 430a999. [](commit_id = 430a999ec9d0f24d960a93580083c996a4db158b, deletion_comment = False)

**Comment by abgoswam** [comment]
>         ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PoissonRegressionWithOptions.cs)]

fix this too ?

---
Refers to: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs:596 in 430a999. [](commit_id = 430a999ec9d0f24d960a93580083c996a4db158b, deletion_comment = False)

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs [hunk 1]
```diff
@@ -567,7 +567,7 @@ public static LbfgsLogisticRegressionBinaryTrainer LbfgsLogisticRegression(this
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PoissonRegression.cs)]
+        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LbfgsPoissonRegression.cs)]
```
Comment (abgoswam) [inline src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: >PoissonRegression [](start = 27, length = 17)

?

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs [hunk 2]
```diff
@@ -567,7 +567,7 @@ public static LbfgsLogisticRegressionBinaryTrainer LbfgsLogisticRegression(this
...
         /// ]]></format>
         /// </example>
         public static LbfgsPoissonRegressionTrainer LbfgsPoissonRegression(this RegressionCatalog.RegressionTrainers catalog,
```
Comment (abgoswam) [inline src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: >LbfgsPoissonRegressionTrainer [](start = 22, length = 29)

there were other renamings which accompanied this .. should we fix those too in this PR ?

[https://github.com/dotnet/machinelearning/issues/3016#issuecomment-476369026](https://github.com/dotnet/machinelearning/issues/3016#issuecomment-476369026)

File: docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj [hunk 3]
```diff
@@ -98,9 +98,9 @@
       <Generator>TextTemplatingFileGenerator</Generator>
       <LastGenOutput>OnlineGradientDescentWithOptions.cs</LastGenOutput>
     </None>
-    <None Update="Dynamic\Trainers\Regression\PoissonRegression.tt">
+    <None Update="Dynamic\Trainers\Regression\LbfgsPoissonRegression.tt">
       <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>PoissonRegression.cs</LastGenOutput>
+      <LastGenOutput>LbfgsPoissonRegression.cs</LastGenOutput>
     </None>
     <None Update="Dynamic\Trainers\Regression\PoissonRegressionWithOptions.tt">
```
Comment (shmoradims) [inline docs/samples/Microsoft.ML.Samples/Microsoft.ML.Samples.csproj]: >PoissonRegressionWithOptions [](start = 46, length = 28)

please also fix this in the same way

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs (hunk 1 above)
Comment (zeahmed) [reply src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: I don't think we need to prefix this with Lbfgs.

---
In reply to: [270581846](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270581846) [](ancestors = 270581846)

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs (hunk 2 above)
Comment (zeahmed) [reply src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: Rest of the examples are not using .tt or they have been converted to .tt after the above change. So, those are fine right now.

---
In reply to: [270582198](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270582198) [](ancestors = 270582198)

File: src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs (hunk 1 above)
Comment (rogancarr) [reply src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs]: This is the alt-text for the link, so it should be the name of the API.

---
In reply to: [270973957](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270973957) [](ancestors = 270973957,270581846)

**Comment by zeahmed** [comment]
Thanks!

**Pull request merged**